include/ece.h

```diff
@@ -93,6 +103,31 @@ ece_aes128gcm_decrypt(
   // caller takes ownership of the buffer, and should free it when it's done.
   ece_buf_t* plaintext);
 
+// Returns the maximum encrypted payload size. The caller should allocate and
+// pass a buffer of this size to `ece_aes128gcm_encrypt*`.
+size_t
+ece_aes128gcm_max_payload_length(uint32_t rs, size_t padLen,
```

Comment:
I found this pattern while reading through the [Rust FFI docs](https://doc.rust-lang.org/book/ffi.html). It seems like the convention in C is to have the caller allocate and pass a pointer to the memory. We also want to give decryption the same treatment.